odriverobotics/ODrive
# Store configuration in non-volatile memory

Use the last two flash pages to store configuration data across reboots.

To test:
```
>>> my_odrive.motor0.pos_gain
20.0
>>> my_odrive.motor0.pos_gain = 30
>>> my_odrive.save_config()
[power cycle ODrive]
>>> my_odrive.motor0.pos_gain
30.0
```

 ~~- [ ] check whether the modifications to the linker script are in conflict with the STM code generator~~ this is easier to check on the `sam_cubemx_cleanup` branch once this one is merged
